config/default.yaml

```diff
@@ -120,4 +120,5 @@ logging:
     baseUrl: https://s3-us-west-2.amazonaws.com/logs.screwdriver.cd
 
 ecosystem:
+    # URL for the User Interface
     ui: http://cd.screwdriver.cd
```

Comment:
We COULD default it.  But I changed it here instead: https://github.com/screwdriver-cd/kubernetes/pull/8